src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -921,4 +921,7 @@ nuget client-certs List</value>
   <data name="Option_Help" xml:space="preserve">
     <value>Show command help and usage information.</value>
   </data>
+  <data name="SourcesCommandProtocolVersionDescription" xml:space="preserve">
+    <value>The NuGet server protocol version to be used. The current version is "3". Supported in NuGet 3.0+. Defaults to "2" if not specified.</value>
```

Comment:
I don't understand it says currently version is `3`, same time it defaults to `2`. 
Do you mean `3` current effective version used by server team? Please clarify further.

Reply:
I've tried to use wording similar to what is said in the table here <https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file#package-source-sections>.

Any suggestions to make it clearer are welcome.

Reply:
I've pushed an update to the description which should be clearer.